modules/ROOT/pages/feature-flagging.adoc

```diff
@@ -44,88 +44,159 @@ The following table shows the available feature flags, a description of their fu
 [%header%autowidth.stretch,cols="a,a"]
 |===
 |Feature Flag | Description
-<.^|`HONOUR_RESERVED_PROPERTIES_PROPERTY`
-|When enabled, reserved properties such as `app.name` can't be overridden by global properties (overrides are ignored).
+<.^|`mule.batch.fixed.aggregator.transaction.record.buffer`
+|When enabled, batch aggregators with fixed size aggregators commit only when a full block is processed.
 
 *Available Since*
 
 * 4.4.0
-* 4.3.0-202102
+
+* 4.3.0-202103
+
+* 4.2.2-202103
 
 *Enabled by Default Since*
 
-* 4.4.0
-* 4.3.1
+* Not enabled by default in any Mule version
 
 *Issue ID*
 
-* MULE-17659
-<.^|`HANDLE_SPLITTER_EXCEPTION_PROPERTY`
-|When enabled, DataWeave correctly handles internal exceptions while splitting a payload, preventing subsequent serialization errors.
+* MULE-19218
+<.^|`mule.compute.connection.errors.in.stats`
+|When enabled, connection errors are computed as part of alerts triggering.
 
 *Available Since*
 
 * 4.4.0
 
+* 4.3.0-202103
+
 *Enabled by Default Since*
 
 * 4.4.0
 
 *Issue ID*
 
-* DW-383
-<.^|`BATCH_FIXED_AGGREGATOR_TRANSACTION_RECORD_BUFFER_PROPERTY`
-|When enabled, batch aggregators with fixed size aggregators commit only when a full block is processed.
+* MULE-19020
+<.^|`mule.enable.default.errorhandler.not.rollback.incorrect.tx`
+|When enabled, the default error handler added by the Runtime doesn't rollback a transaction if it doesn't correspond.
 
 *Available Since*
 
-* 4.4.0
-* 4.3.0-202103
-* 4.2.2-202103
+* 4.5.0
+
+* 4.4.0-202201
+
+* 4.3.0-202201
 
 *Enabled by Default Since*
 
-* Not enabled by default in any Mule version
+* 4.5.0
+
+* 4.4.1
+
+* 4.3.1
 
 *Issue ID*
 
-* MULE-19218
-<.^|`COMPUTE_CONNECTION_ERRORS_IN_STATS_PROPERTY`
-|When enabled, connection errors are computed as part of alerts triggering.
+* MULE-19919
```

Comment:
Are you sure that you used the latest version of the [MuleRuntimeFeature.java](https://github.com/mulesoft/mule-api/blob/master/src/main/java/org/mule/runtime/api/config/MuleRuntimeFeature.java)?

This one should list more versions than 4.5.0. Can you make another check at the other ones also and submit again?

Reply:
The latest version of the document only lists the v4.5.0